### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by realwebdev**
code review.

### CODE REVIEW

File: database/conn.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		log.Print("Error occured While Deleting Previous Table")
+		return err
+	}
+	log.Print("Previous Table drop successfully")
+
+	if err := db.AutoMigrate(modelObject).Error; err != nil {
+		return err
+	}
+	log.Print("models created successfully")
+	return nil
```
Comment (Bilaltanveer38) [inline database/conn.go:23]: line break above this

File: models/project.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+)
+
+type Project struct {
+	Activitys    []Activity //`gorm:"ForeignKey:ProjectID"`
+	ID           uint
+	UserID       uint
+	Project_name string
+}
+
+func CreateProject(proj Project, db *gorm.DB) {
```
Comment (Bilaltanveer38) [inline models/project.go:16]: project

File: models/project.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	if err := db.Create(&proj).Error; err != nil {
+		log.Print("Error Occured while creating project!")
+		return
+	}
+	log.Print("project has been created")
+}
+
+func UpdateProject(project_id uint, update_name interface{}, db *gorm.DB) {
```
Comment (Bilaltanveer38) [inline models/project.go:24]: func UpdateProject(project_id uint, updates map[string] interface{}, db *gorm.DB) {

**Comment by realwebdev** [comment]
complete

**Pull request closed without merging**